treehouses/remote
# refactor encryptor.kt (fixes #1182)

To test:

1. Go to ssh (beta) fragment
2. Tap on generate key and fill in form

Keys should be generated without any issue
